swarm/OWNERS

```diff
@@ -0,0 +1,26 @@
+# Ownership by go packages
+
+swarm
+├── api ─────────────────── ethersphere
+├── bmt ─────────────────── @zelig
+├── dev ─────────────────── @lmars
+├── fuse ────────────────── @jmozah, @holisticode
+├── grafana_dashboards ──── @nonsense
+├── metrics ─────────────── @nonsense, @holisticode
+├── multihash ───────────── @nolash
+├── network ─────────────── ethersphere
+│   ├── bitvector ───────── @zelig, @janos, @gbalint
+│   ├── priorityqueue ───── @zelig, @janos, @gbalint
+│   ├── simulations ─────── @zelig
+│   └── stream ──────────── @janos, @zelig, @gbalint, @holisticode, @justelad
+│       ├── intervals ───── @janos
+│       └── testing ─────── @zelig
+├── pot ─────────────────── @zelig
+├── pss ─────────────────── @nolash, @zelig, @nonsense
+├── services ────────────── @zelig
+├── state ───────────────── @justelad
+├── storage ─────────────── ethersphere
+│   ├── encryption ──────── @gbalint, @zelig, @nagydani
+│   ├── mock ────────────── @janos
+│   └── mru ─────────────── @nolash
+└── testutil ────────────── @lmars
```

Comment:
If you merged the contents of this file to `.github/CODEOWNERS`, the correct person also get a review request every time someone modifies these packages.

Reply:
Great idea, done in https://github.com/ethersphere/go-ethereum/commit/ababfd22bc93619d6197846d2e2d74fb0aea6f4e